graylog2-server/src/main/java/org/graylog2/shared/bindings/providers/ProxiedRequestsExecutorService.java

```diff
@@ -27,24 +29,33 @@
 import javax.inject.Singleton;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+
+import static com.codahale.metrics.MetricRegistry.name;
 
 @Singleton
 public class ProxiedRequestsExecutorService implements Provider<ExecutorService> {
     private final int proxiedRequestsMaxThreads;
+    private final MetricRegistry metricRegistry;
 
     @Inject
-    public ProxiedRequestsExecutorService(@Named("proxied_requests_thread_pool_size") int proxiedRequestsMaxThreads) {
+    public ProxiedRequestsExecutorService(@Named("proxied_requests_thread_pool_size") int proxiedRequestsMaxThreads,
+                                          MetricRegistry metricRegistry) {
         this.proxiedRequestsMaxThreads = proxiedRequestsMaxThreads;
+        this.metricRegistry = metricRegistry;
     }
 
     @Override
     public ExecutorService get() {
-        return Executors.newFixedThreadPool(proxiedRequestsMaxThreads,
-            new ThreadFactoryBuilder()
+        final ThreadFactory threadFactory = new ThreadFactoryBuilder()
                 .setNameFormat("proxied-requests-pool-%d")
                 .setDaemon(true)
-                .setUncaughtExceptionHandler(new Tools.LogUncaughtExceptionHandler(LoggerFactory.getLogger(ProxiedResource.class.getName())))
-                .build()
-        );
+                .setUncaughtExceptionHandler(
+                        new Tools.LogUncaughtExceptionHandler(LoggerFactory.getLogger(ProxiedResource.class.getName())))
+                .build();
+        return new InstrumentedExecutorService(
+                Executors.newFixedThreadPool(proxiedRequestsMaxThreads, threadFactory),
+                metricRegistry,
+                name(this.getClass()));
```

Comment:
Let's use a name for the metric in addition to the class name. (like we do in [JerseyService](https://github.com/Graylog2/graylog2-server/blob/dba006b663632f6265882328d3134c7cb03c3456/graylog2-server/src/main/java/org/graylog2/shared/initializers/JerseyService.java#L325)) Like `http-proxied-requests-executor`? What do you think?
```suggestion
                name(this.getClass(), "http-proxied-requests-executor"));
```
If we do that, we also need to update the Prometheus exporter mapping file.